packages/apps/composer-app/socket.ini

```diff
@@ -202,37 +201,39 @@ icon_sizes = "512@1x"
 [ios]
 
 ; signing guide: https://socketsupply.co/guides/#ios-1
-codesign_identity = ""
+codesign_identity = "iPhone Developer: Rich Burdon (8VF4CF6T8M)"
 
 ; Describes how Xcode should export the archive. Available options: app-store, package, release-testing, enterprise, development, and developer-id.
-distribution_method = "release-testing"
+; distribution_method = "release-testing"
+distribution_method = "development"
 
 ; A path to the provisioning profile used for signing iOS app.
-provisioning_profile = ""
+provisioning_profile = "profiles/burdoncomposer.mobileprovision"
 
 ; which device to target when building for the simulator.
-simulator_device = "iPhone 14"
+simulator_device = "iPad Pro 11"
```

Comment:
Ditto here, I imagine we could have project defaults, but these provisioning certificates are sensitive.